workshop/Zadanie 2/README.md

```diff
@@ -0,0 +1,62 @@
+# Zadanie 2 - Czytaj więcej
+
+## Jak pracować z instrukcjami?
+
+1. Dokładnie zapoznaj się z każdym krokiem zadania.
+2. Na bieżąco wprowadzaj zmiany do swojego kodu zgodnie z instrukcjami.
+3. Jeśli coś jest niejasne, pytaj śmiało!
+
+## Cel zadania
+
+Naszym celem jest sprawienie, by po kliknięciu w przycisk `Więcej` rozwijał się opis pod daną sekcją. Jednocześnie przycisk ma zamienić się na `Mniej` i po kliknięciu chować pokazany wcześniej tekst.
+
+## Instrukcja
+
+1. Poszukaj w kodzie HTML przycisków `Więcej`. Nad każdym z nich widzisz teraz dwa paragrafy. Jeden z nich ma klasę `more`. Ukryj wszystkie paragrafy o klasie `more` przez zmianę w `display` na `none` w pliku CSS. Nastepnie w pliku JS zapisz wszystkie przyciski `Więcej` do zmiennej używając `document.querySelectorAll('nazwa-selektora'`). Wyświetl zmienną w konsoli - powinnaś zobaczyć tablicę z trzema przyciskami.
+2. Nad każdym przyciskiem znajduje się paragraf o klasie `more` (mimo że teraz jest niewidoczny). Przycisk oraz paragraf mają tego samego rodzica, którym jest nadrzędny dla nich `section`. Są więc rodzeństwem. W JS możemy to wykorzystać. Po kliknięciu w przycisk ma pojawiać się jego poprzedni brat (albo siostra ;)). Żeby sprawdzić, czy faktycznie tak to zadziała, wypisz w konsoli rodzeństwo pierwszego elementu tablicy z przyciskami w ten sposób:
+```javascript
+    console.log(buttons[0].previousElementSibling);
```

Comment:
Pojawia się nam magiczna zmienna ```buttons``` której nigdy wcześniej nie było - jeżeli chcesz się do niej odwoływać to w poprzednim punkcie musisz dodać snippet z tworzeniem tej zmiennej i wrzuceniem do niej wyszukanych guzików.